crossbar/controller/processtypes.py

```diff
@@ -117,10 +117,21 @@ def _dump_remaining_log(self, result):
 
     def log(self, childFD, data):
         """
-        FIXME: line buffering
+        Handle a log message (or a fragment of such) coming in.
         """
         assert(childFD in self._log_fds)
 
+        system = "{:<10} {:>6}".format(self.LOGNAME, self.pid)
+
+        if childFD == 1:
```

Comment:
FD2 is our special JSON stuff right now -- but maybe it should have its own pipe too, yeah.